tests/UnitTests/TemplateSource/TagTests/PluginModifier/PluginModifierExplodeTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace UnitTests\TemplateSource\TagTests\PluginModifier;
+
+/**
+ * class for modifier tests
+ *
+ * @runTestsInSeparateProcess
+ * @preserveGlobalState disabled
+ * @backupStaticAttributes enabled
+ */
+class PluginModifierExplodeTest extends \PHPUnit_Smarty
+{
+    public function setUp(): void
+    {
+        $this->setUpSmarty(__DIR__);
+    }
+
+    /**
+     * @return void
+     * @throws \SmartyException
+     *
+     * @dataProvider explodeDataProvider
+     */
+    public function testExplode($template, $subject, $expectedString)
+    {
+        $this->smarty->setDebugging(true);
```

Comment:
> This enables the [debugging console](https://smarty-php.github.io/smarty/programmers/api-variables/variable-debugging.html#chapter.debugging.console). The console is a javascript popup window that informs you of the [included](https://smarty-php.github.io/smarty/programmers/api-variables/variable-debugging.html#language.function.include) templates, variables [assigned](https://smarty-php.github.io/smarty/programmers/api-variables/variable-debugging.html#api.assign) from php and [config file variables](https://smarty-php.github.io/smarty/programmers/api-variables/variable-debugging.html#language.config.variables) for the current script.

Why enable the debugging console in unit tests?

Reply:
I don't remember what I was trying to accomplish with that line. I've removed it now